package.json

```diff
@@ -2,19 +2,37 @@
   "name": "sdkv2",
   "version": "0.1.0",
   "license": "Apache-2.0",
-  "workspaces": [
-    "1-base-layer",
-    "2-definition-layer",
-    "3-platform-layer/*",
-    "4-xdapp-layer"
-  ],
   "devDependencies": {
     "@types/chai": "^4.3.5",
+    "@types/jest": "^29.5.3",
     "@types/mocha": "^10.0.1",
     "@types/node": "^20.4.1",
     "chai": "^4.3.7",
+    "jest": "^29.6.1",
     "mocha": "^10.2.0",
+    "ts-jest": "^29.1.1",
     "ts-mocha": "^10.0.0",
+    "ts-node": "^10.9.1",
     "typescript": "^5.1.6"
-  }
+  },
+  "scripts": {
+    "lint": "npm run lint --workspaces --if-present",
+    "build": "npm run build --workspaces --if-present",
+    "compile": "npm run compile --workspaces --if-present",
+    "test": "npm run test --workspaces --if-present",
+    "coverage": "npm run coverage --workspaces --if-present"
+  },
+  "workspaces": [
+    "1-base-layer",
+    "2-definition-layer",
+    "3-platform-layer/*",
+    "4-xdapp-layer/*",
```

Comment:
should the above have `package/` path? This does seem to work fwiw

Reply:
I missed these when I moved everything into the packages directory, thanks! Fixed now